src/main/java/org/dcsa/bkg/service/impl/BookingServiceImpl.java

```diff
@@ -1360,18 +1364,25 @@ The voyage number on a booking is always the Export voyage number (the shipment
                 Mono.zip(
                     bookingRepository
                         .updateDocumentStatusAndUpdatedDateTimeForCarrierBookingRequestReference(
-                            DocumentStatus.CANC,
-                            carrierBookingRequestReference,
-                            OffsetDateTime.now())
+                            DocumentStatus.CANC, carrierBookingRequestReference, updatedDateTime)
```

Comment:
I thought it was decided to let the db create the timestamp

Reply:
I think it is a good idea that the ShipmentEvent and the booking "creationDateTimes" are the same (I know it is not a requirement)

Reply:
@pimg I thought we agreed on keeping it as is, but I could be wrong. Makes it easier, at least in this case, for returning a response that includes the updated time without making another query.

Reply:
I think saving the query makes a lot of sense, so indeed leaving it would be fine